src/components/DocsToc.js

```diff
@@ -22,17 +22,50 @@ const additionalStyles = {
   }
 };
 
+const baseIndent = 15;
+const baseFontSize = 22;
+
+const Menu = ({ indent, menu }) => {
+  const children = useMenus({ filter: m => m.parent === menu.name });
+  const style = {
+    paddingLeft: indent * baseIndent,
+    fontSize: baseFontSize - indent * 4 - indent,
+    listStyleType: 'none',
+  };
+  if (children) {
+    return (
+      <li style={style}>
+        <MenuLink to={menu.route} label={menu.name} />
+        <ul style={style}>
+          {children.map(m => (
+            <Menu
+              key={m.id}
+              indent={indent + 1}
+              menu={m}
+            />
+          ))}
+        </ul>
+      </li>
+    );
+  } else {
+    return (
+      <li key={menu.id} style={style}>
+        <MenuLink to={menu.route} label={menu.name} />
+      </li>
+    );
+  }
+}
+
 const DocsToc = ({ classes }) => {
-  const toc = groupByParent(useMenus({ filter: m => m.route.startsWith('/docs/')}))
+  const [query] = useState('');
```

Comment:
I know this is copied but it just seems useless to me. Can we try whether the current behavior is preserved without this state variable and evaluating the ternary below always to false?

If it still works please also fix it where this is coming from.

Reply:
I removed query